backend/src/main/java/ch/ase21/backend/service/NeighbourhoodService.java

```diff
@@ -78,9 +78,9 @@ private NeighbourhoodService() {/* void */}
     double breakEven =  averagePricePerApartment / (averageRevenuePerNight * 365);
     double score = 7 / breakEven; // Score of 1.0 if break even is in 7 years or less
     if(score > 1.0){
-      score = 1.0;
+      score = 10.0;
     } else {
-      score = Math.round(score * 10.0) / 10.0;
+      score = Math.round(score * 100.0) / 10.0;
```

Comment:
I would adjust this to just `Math.round(score * 10)`, otherwise you might end up with scores like 5.5, which the frontend will round anyways

Reply:
I need the more accurate scores for the overall rating.